.php_cs.dist

```diff
@@ -27,6 +27,7 @@ $config
         '@PHPUnit75Migration:risky' => true,
         '@PhpCsFixer' => true,
         '@PhpCsFixer:risky' => true,
+        'general_phpdoc_annotation_remove' => ['annotations' => ['expectedDeprecation']],
```

Comment:
can you create a feature request to autoconvert annotation to method call?

Reply:
done: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5616